<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSavingDialog</class>
 <widget class="QDialog" name="AddSavingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddSavingDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(35, 39, 42);

</string>
  </property>
  <widget class="QLineEdit" name="savingTitleLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit[text]{
color: white;}
QLineEdit{
background-color: rgb(44, 47, 51);
border:none;}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Saving Title</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="savingGoalLineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit[text]{
color: white;}
QLineEdit{
background-color: rgb(44, 47, 51);
border:none;}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Saving Goal</string>
   </property>
  </widget>
  <widget class="QLabel" name="isoLabel">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>270</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel[text]{
color: white;}
QLabel{
background-color:rgb(35, 39, 42);
border:none;}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addSavingPushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>420</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
     background-color: rgb(88, 101, 242);
 }
 QPushButton#pushButton:pressed {
   	background-color: rgb(88, 157, 242); 
 }
 QPushButton#pushButton:hover {
     
	background-color: rgb(88, 157, 242);
 }
 QPushButton[text] {
    color:white;
 }</string>
   </property>
   <property name="text">
    <string>Add Saving</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>420</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
     background-color: rgb(88, 101, 242);
 }
 QPushButton#pushButton:pressed {
   	background-color: rgb(88, 157, 242); 
 }
 QPushButton#pushButton:hover {
     
	background-color: rgb(88, 157, 242);
 }
 QPushButton[text] {
    color:white;
 }</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>AddSavingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>504</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>488</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
